.github/workflows/ci.yml

```
@@ -405,16 +405,16 @@ jobs:
         env:
           PKG_CONFIG_PATH: /usr/lib/x86_64-linux-gnu/pkgconfig
 
-  build-gcc11:
-    name: Build on gcc 11
-    if: contains(github.head_ref, 'dist') || contains(github.head_ref, 'gcc11') || contains(github.ref, 'release-') || github.ref == 'refs/heads/stable' || github.event_name == 'schedule'
+  build-gcc12:
+    name: Build on gcc 12
+    if: contains(github.head_ref, 'dist') || contains(github.head_ref, 'gcc12') || contains(github.ref, 'release-') || github.ref == 'refs/heads/stable' || github.event_name == 'schedule'
     runs-on: ubuntu-latest
-    container: ubuntu:hirsute
+    container: ubuntu:kinetic
```

```suggestion
    container: ubuntu:jammy
```

I think Ubuntu Jammy (22.04) is better since it has LTS and therefore won't go EOL anytime soon.

---

Ok